Update tasks

Fischmiep opened the pull request:
Some phrasing in the exercises had a some German accent. I tried to improve/fix it to the best of my ability.

Commit:
Update README

Commit:
Revert Resources heading levels

Commit:
Made exercise descriptions a little nicer

esrehmki commented:
```
 root_dir = ''
 if pathlib.Path.cwd().name == 'python-for-chemists':
   root_dir = pathlib.Path.cwd()
 else:
   root_dir = pathlib.Path.cwd().parent
 
 data_path = root_dir / 'Data'
 data_file = data_path / 'input_for_loop_exercise.txt'

```
You removed the conditional root node determination. The tests need a different root node than the notebooks. this fixes it for me

Fischmiep committed:
Fix rootdir setup for loops exercise

The tests need a different root directory than the notebooks. This fixes a failing pytest.

Fischmiep commented:
> ```
>  root_dir = ''
>  if pathlib.Path.cwd().name == 'python-for-chemists':
>    root_dir = pathlib.Path.cwd()
>  else:
>    root_dir = pathlib.Path.cwd().parent
>  
>  data_path = root_dir / 'Data'
>  data_file = data_path / 'input_for_loop_exercise.txt'
> ```
> 
> You removed the conditional root node determination. The tests need a different root node than the notebooks. this fixes it for me

You're right. I copied the code from the Exercise which is missing this condition. Should be fixed now
